### PR TITLE
174 webinar related content

### LINKED ISSUE #174: [Webinar] Detail Page(Logged User)
- [x] Back to Navigation
- [x] Column[Reuse]
- [x] Title[Reuse]
- [x] Text[Reuse]
- [x] Learn More
- [x] Columns(More Episode)
- [x] RelatedStory (You may be interseted in)

Figma Designs: https://www.figma.com/design/0MPkm5mzMfI0UPS6DWdbKs/Optimus-%2F-Abcam-Website-Designs?node-id=2030-11781&node-type=section&t=oiOLqNYEnznjpWOF-0

### PR DESCRIPTION
**Pull request by venkateshamkeerthi**
changes done for webinar related-stories but this can be tested after indexing data updated for webinar,
this component is used for stories also, we are testing stories pages also, shouldn't impact other pages

Fix #174 

Test URLs:
- Before: https://main--danaher-abcam--aemsites.hlx.live/en-us/stories/articles/simplify-your-assay-design-with-our-matched-antibody-pairs
- After:  https://174-webinar-related-content--danaher-abcam--aemsites.hlx.live/en-us/stories/articles/simplify-your-assay-design-with-our-matched-antibody-pairs

### CODE REVIEW

File: eds/blocks/related-stories/related-stories.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ function createCard(article, firstCard = false) {
     : article.title;
```
Comment (rgravitvl) [inline eds/blocks/related-stories/related-stories.js:11]: why don't we use createCard from articleCard.js here??

**Review by rgravitvl — CHANGES_REQUESTED** [review]
why don't we use createCard from articleCard.js here??

File: eds/blocks/related-stories/related-stories.js (hunk 1 above)
Comment (venkateshamkeerthi) [reply eds/blocks/related-stories/related-stories.js:11]: styling is different for hub page and related--stories cards(https://174-webinar-related-content--danaher-abcam--aemsites.hlx.live/en-us/stories/articles/simplify-your-assay-design-with-our-matched-antibody-pairs)
ex: https://174-webinar-related-content--danaher-abcam--aemsites.hlx.live/en-us/stories and
https://174-webinar-related-content--danaher-abcam--aemsites.hlx.live/en-us/stories/articles/simplify-your-assay-design-with-our-matched-antibody-pairs

"Read Article"  link and on mouseover image scale-up(zoom)

Comment (venkateshamkeerthi) [reply eds/blocks/related-stories/related-stories.js]: Hi @rgravitvl  I changed logic changed to  createCard from articleCard.js

**Review by rgravitvl — APPROVED** [review]

**Review by pardeepgera23 — APPROVED** [review]

**Pull request merged**